### PR TITLE
CORE-6223: Rename x500name to x500Name in the APIs

### PR DESCRIPTION
**Pull request by yift-r3**
Changing the argument name in the P2P tool and API to be consistent. We need to change the wiki after this is merged.

### CODE REVIEW

**Review by vkolomeyko — DISMISSED** [review]
LGTM (on behalf of @corda/http-rpc)

File: applications/tools/flow-worker-setup/README.md [hunk 1]
```diff
@@ -79,15 +79,15 @@ To publish the configuration required by the `LinkManager` which is used by the
   ],
   "membersToAdd": [
     {
-      "x500name": "CN=Alice, O=Alice Corp, L=LDN, C=GB",
+        "x500Name": "CN=Alice, O=Alice Corp, L=LDN, C=GB",
```
Comment (nikinagy) [inline applications/tools/flow-worker-setup/README.md]: Remove the extra space please

**Review by nikinagy — APPROVED** [review]
LGTM

**Review by williamvigorr3 — APPROVED** [review]
LGTM

**Review by benmcmahon100 — APPROVED** [review]

**Pull request merged**